xapian-core/queryparser/queryparser.lt

```diff
@@ -569,16 +569,14 @@ static void yyStackOverflow(yyParser *yypParser){
 */
 #ifdef XAPIAN_DEBUG_LOG
 static void yyTraceShift(yyParser *yypParser, int yyNewState, const char *zTag){
-  if( yyTraceFILE ){
-    if( yyNewState<YYNSTATE ){
-      LOGLINE(QUERYPARSER, zTag << " '" <<
-			   yyTokenName[yypParser->yystack.back().major] <<
-			   "', go to state " << yyNewState);
-    }else{
-      LOGLINE(QUERYPARSER, zTag << " '" <<
-			   yyTokenName[yypParser->yystack.back().major] <<
-			   "', pending reduce " << yyNewState - YY_MIN_REDUCE);
-    }
+  if( yyNewState<YYNSTATE ){
+    LOGLINE(QUERYPARSER, zTag << " '" <<
+     yyTokenName[yypParser->yystack.back().major] <<
+     "', go to state " << yyNewState);
+  }else{
+    LOGLINE(QUERYPARSER, zTag << " '" <<
+     yyTokenName[yypParser->yystack.back().major] <<
+     "', pending reduce " << yyNewState - YY_MIN_REDUCE);
```

Comment:
This code has changed, but also has similar indentation problems on the continuation lines now.  The first non-whitespace character of the continuation lines should line up under the `z` of `zTag` as it did before (in an editor at least; in github's diff viewer it appears to be off by one column because the first line a 4 space indent, but the continuation lines start with a tab, and so the leading `-` makes a difference).

Reply:
Addressed in comment above.